### PR TITLE
Ref Task: 6229 Get data from API for discover screen part 2

### PR DESCRIPTION
**Pull request by trieuthan92pro**
Link redmine: https://edu-redmine.framgia.vn/issues/6229

### CODE REVIEW

**Comment by trieuthan92pro** [comment]
Ready

**Comment by DoanVanToan** [comment]
Pull 8 đựoc merged thì a rebase r gửi pull 9 a nhé

**Comment by trieuthan92pro** [comment]
Ready

File: .gitignore [hunk 1]
```diff
@@ -43,7 +43,7 @@ captures/
 .idea/caches
 
 # Keystore files
-# Uncomment the following line if you do not want to check your keystore files in.
+# Uncomment the following line if you do not want to favorite your keystore files in.
```
Comment (DoanVanToan) [inline .gitignore:46]: keep old code.

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**